packages/common/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "abab": "^2.0.5",
     "adlib": "^3.0.7",
+    "jsonapi-typescript": "^0.1.3",
```

Comment:
This should be a dev dep, right?

Reply:
good question. I'm not sure since we export a type that is based on a `jssonapi-typescript` type. I've been in these debates before, and essentially landed at - it doesn't hurt to have it as a dependency.